On musl libc, time_t is a 64-bit type. By default, ChezScheme assumes
that time_t is a 32-bit type on x86. This patch adjusts the ChezScheme
configuration files accordingly.

See also: https://github.com/cisco/ChezScheme/issues/527

diff -upr csv9.5.8.orig/boot/ti3le/equates.h csv9.5.8/boot/ti3le/equates.h
--- csv9.5.8.orig/boot/ti3le/equates.h	2022-04-30 15:30:23.580730631 +0200
+++ csv9.5.8/boot/ti3le/equates.h	2022-04-30 15:30:34.110815342 +0200
@@ -629,7 +629,7 @@ typedef unsigned long long U64;
 #define time_duration 0x2
 #define time_monotonic 0x3
 #define time_process 0x0
-#define time_t_bits 0x20
+#define time_t_bits 0x40
 #define time_thread 0x1
 #define time_utc 0x4
 #define timer_interrupt_index 0x2
diff -upr csv9.5.8.orig/s/ti3le.def csv9.5.8/s/ti3le.def
--- csv9.5.8.orig/s/ti3le.def	2022-04-30 15:30:23.607397512 +0200
+++ csv9.5.8/s/ti3le.def	2022-04-30 15:30:34.107481982 +0200
@@ -24,7 +24,7 @@
 (define-constant size_t-bits 32)
 (define-constant ptrdiff_t-bits 32)
 (define-constant wchar-bits 32)
-(define-constant time-t-bits 32)
+(define-constant time-t-bits 64)
 (define-constant max-float-alignment 4)
 (define-constant max-integer-alignment 4)
 (define-constant asm-arg-reg-max 1)
